test_scripts/Resumption/Handling_errors_from_HMI/020_resume_all_data_with_one_error_rpc_unexpected_disconnect_limited.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Requirement summary:TBD
+--
+-- Description:
+-- In case:
+-- 1. App is in Limited HMI level
+-- 2. Rpc_n for resumption is added by app
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App reregisters with actual HashId
+-- 5. Rpc_n request is sent from SDL to HMI during resumption
+-- 6. HMI responds with error resultCode to Rpc_n request
+-- SDL does:
+-- 1. process unsuccess response from HMI
+-- 2. respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application
+-- 3. rrestore Limited HMI level
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Common Functions ]]
+local function deactivateAppToLimited()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", {
+      appID = common.getHMIAppId()
+    })
+  common.getMobileSession():ExpectNotification("OnHMIStatus",
+    {hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+end
+
```

Comment:
@GetmanetsIrina Propose to move `deactivateAppToLimited()` function to `common` module since it's used in a few scripts